redwood-basic-view/src/main/kotlin/app/cash/redwood/basic/view/ViewImage.kt

```diff
@@ -34,7 +37,10 @@ internal class ViewImage(
   override var modifier: Modifier = Modifier
 
   override fun url(url: String) {
-    value.load(url)
+    val request = ImageRequest.Builder(value.context)
+      .target(value)
```

Comment:
Missing `.data(url)`:

```suggestion
      .target(value)
      .data(url)
```

Reply:
Whoops. Yeah, that's somewhat important.